charts/castai-evictor/values.yaml

```diff
@@ -93,13 +93,10 @@ affinity:
     requiredDuringSchedulingIgnoredDuringExecution:
       nodeSelectorTerms:
         - matchExpressions:
-            - key: "kubernetes.io/os"
-              operator: In
-              values: ["linux"]
-        - matchExpressions:
-            - key: "beta.kubernetes.io/os"
-              operator: In
-              values: ["linux"]
+            - key: kubernetes.io/os
```

Comment:
wasn't the explicit requirement better? E.g. we would match `beta.kubernetes.io/os=windows" with this.

Reply:
The initial requirement was just to ignore windows nodes and 99% runs on linux. And `beta.kube.io/os=linux` is deprecated so we now always get warnings during components install, upgrade.